cns/client/client_test.go

```diff
@@ -391,3 +392,39 @@ func TestCNSClientDebugAPI(t *testing.T) {
 	t.Logf("PodIPConfigState: %+v", inmemory.HTTPRestServiceData.PodIPConfigState)
 	t.Logf("IPAMPoolMonitor: %+v", inmemory.HTTPRestServiceData.IPAMPoolMonitor)
 }
+
+func TestNew(t *testing.T) {
+	emptyRoutes, _ := buildRoutes(defaultBaseURL, clientPaths)
+	tests := []struct {
+		name    string
+		url     string
+		timeout time.Duration
+		want    *Client
+		wantErr bool
+	}{
+		{
+			name: "empty url",
+			url:  "",
+			want: &Client{
+				routes: emptyRoutes,
+				client: http.Client{
+					Timeout: 0,
```

Comment:
is this line needed?

Reply:
strictly speaking no, that's the zero val